docs/content/manual/manual.yml

```diff
@@ -2944,13 +2946,15 @@ sections:
 
           Outputs one new input.
 
+          Commonly used in combination with the `-n` option.
+
       - title: "`inputs`"
         body: |
 
           Outputs all remaining inputs, one by one.
 
           This is primarily useful for reductions over a program's
-          inputs.
+          inputs.  Commonly used in combination with the `-n` option.
```

Comment:
I think this should be worded more strongly. If you omit `-n`, you lose the first item of the stream. That almost definitely produces incorrect results.

Perhaps you could quote @pkoppstein 's excellent [A stream oriented introduction to jq](https://github.com/pkoppstein/jq/wiki/A-Stream-oriented-Introduction-to-jq)?

> Note that when using inputs in this way, it will generally be necessary to invoke jq with the -n command-line option, otherwise the first entity will be lost.

Reply:
Good idea, reworded a bit